File: 백준/Bronze/11655. ROT13/ROT13.cs
string input = Console.ReadLine();
string encrypt = "";

for (int i = 0; i < input.Length; i++)
{
    if (char.IsLower(input[i]))
    {
        int rot13 = (input[i] + 13);

        encrypt += rot13 <= 'z' ? (char)rot13 : (char)('a'+ rot13-'z'-1);
    }
    else if (char.IsUpper(input[i]))
    {
        int rot13 = (input[i] + 13);

        encrypt += rot13 <= 'Z' ? (char)rot13 : (char)('A' + rot13-'Z'-1);
    }
    else
    {
        encrypt += input[i];
    }
}

Console.WriteLine(encrypt);

File: 백준/Bronze/11655. ROT13/ROT13.cc
#include <iostream>
#include <string>

using namespace std;

string s;

int main()
{
	getline(cin, s);

	for (int i = 0; i < s.size(); i++)
	{
		if (islower(s[i]))
		{
			s[i] = (char)(s[i] + 13 > 'z' ? s[i] + 13 -26 : s[i]+13);
		}
		else if (isupper(s[i]))
		{
			s[i] = (char)(s[i] + 13 > 'Z' ? s[i] + 13 - 26 : s[i] + 13);
		}
	}

	cout << s;

	return 0;
}